projects/samples/robotbenchmark/highway_driving/plugins/robot_windows/highway_driving/highway_driving.html

```diff
@@ -2,57 +2,62 @@
 <html>
   <head>
     <meta charset='UTF-8'>
+    <link type="text/css" rel="stylesheet" href='../../../../include/tabs.css' />
   </head>
   <body>
-    <div id='infotabs' class='webotsTabs webotsRobotWindowTabs'>
-      <ul>
-        <li><a href='#infotab-overview'>Overview</a></li>
-        <li><a href='#infotab-metrics'>Metrics</a></li>
-        <li><a href='#infotab-instructions'>Instructions</a></li>
-      </ul>
-      <div id='infotab-overview'>
-        <p>
-          This benchmark aims at driving the farther as possible in one minute without colliding any other vehicles or static objects.
-        </p>
+    <div id='infotabs' class='wrapper webotsTabs webotsRobotWindowTabs'>
+      <input class="radio" id="one" name="group" type="radio" checked>
+      <input class="radio" id="two" name="group" type="radio">
+      <input class="radio" id="three" name="group" type="radio">
+      <div class="tabs">
+        <label class="tab" id="one-tab" for="one">Overview</label>
+        <label class="tab" id="two-tab" for="two">Metrics</label>
+        <label class="tab" id="three-tab" for="three">Instructions</label>
```

Comment:
I would rather keep specific ids describing the content (overview, metric, instructions, etc.) instead of using generic "one", "two", "three", etc. that doesn't tell anything about the content.

Same for panel ids, etc.